model/docs/getting-started.qmd

```diff
@@ -154,42 +161,49 @@ ax.set_yscale("log")
 
 `pyrenew` leverages `numpyro`'s flexibility to build models via composition. As a principle, most objects in `pyrenew` can be treated as random variables from which we can sample. At the top-level `pyrenew` has two metaclass from which most objects inherit: `RandomVariable` and `Model`. From them, the following four sub-modules arise:
 
-- The `process` sub-module,
-- The `observation` sub-module,
-- The `latent` sub-module, and
-- The `models` sub-module
+-   The `process` sub-module,
+-   The `deterministic` sub-module,
+-   The `observation` sub-module,
+-   The `latent` sub-module, and
+-   The `models` sub-module
 
-Where the first three are collections of instances of `RandomVariable` and the last one a collection of instances of `Model`. The following diagram shows a detailed view of how meta classes, modules, and classes interact to create the `RtInfectionsRenewalModel` instantiated in the previous section:
+Where the first four are collections of instances of `RandomVariable` and the last one a collection of instances of `Model`. The following diagram shows a detailed view of how meta classes, modules, and classes interact to create the `RtInfectionsRenewalModel` instantiated in the previous section:
 
 ```{mermaid}
 %%| label: overview-of-RtInfectionsRenewalModel
-flowchart TB
+flowchart LR
     rand((RandomVariable\nmetaclass))
     models((Model\nmetaclass))
```

Comment:
Suggest putting the model metaclass above the random variable metaclass in the diagram, so that the metaclasses form a single column

Reply:
I know, but is so wide! 

```mermaid
flowchart TB
    rand((RandomVariable\nmetaclass))
    models((Model\nmetaclass))

    subgraph observations[Observations module]
        obs["observed_infections\n(PoissonObservation)"]
    end

    subgraph latent[Latent module]
        inf["latent_infections\n(Infections)"]
        i0["I0\n(Infections0)"]
    end

    subgraph process[Process module]
        rt["rt_proc\n(RtRandomWalkProcess)"]
    end

    subgraph deterministic[Deterministic module]
        detpmf["gen_int\n(DeterministicPMF)"]
    end

    subgraph model[Model module]
        model1["model1\n(RtInfectionsRenewalModel)"]
    end

    rand-->|Inherited by|observations
    rand-->|Inherited by|process
    rand-->|Inherited by|latent
    rand-->|Inherited by|deterministic
    models-->|Inherited by|model

    detpmf-->|Composes|model1
    i0-->|Composes|model1
    rt-->|Composes|model1
    obs-->|Composes|model1
    inf-->|Composes|model1
```

```mermaid
flowchart LR
    rand((RandomVariable\nmetaclass))
    models((Model\nmetaclass))

    subgraph observations[Observations module]
        obs["observed_infections\n(PoissonObservation)"]
    end

    subgraph latent[Latent module]
        inf["latent_infections\n(Infections)"]
        i0["I0\n(Infections0)"]
    end

    subgraph process[Process module]
        rt["rt_proc\n(RtRandomWalkProcess)"]
    end

    subgraph deterministic[Deterministic module]
        detpmf["gen_int\n(DeterministicPMF)"]
    end

    subgraph model[Model module]
        model1["model1\n(RtInfectionsRenewalModel)"]
    end

    rand-->|Inherited by|observations
    rand-->|Inherited by|process
    rand-->|Inherited by|latent
    rand-->|Inherited by|deterministic
    models-->|Inherited by|model

    detpmf-->|Composes|model1
    i0-->|Composes|model1
    rt-->|Composes|model1
    obs-->|Composes|model1
    inf-->|Composes|model1
```

But if you feel strongly, I can change it!

Reply:
My suggestion was still to use the portrait layout (second of the two here), but just with the circles all on the left (which to avoid overlapping arrows will require `Model metaclass` to go above or below `RandomVariable metaclass`. On reflection, I think above would be best, since its child `Model module` is labeled at the top.

Reply:
For some reason your suggestion was not up to date. But we are on the same page. LR it is!